package.json

```diff
@@ -222,6 +223,9 @@
     "@babel/preset-env": "7.24.4",
     "@babel/traverse": "7.24.1",
     "@babel/types": "7.24.0",
+    "@parcel/transformer-react-refresh-wrap": "2.14.0",
+    "@parcel/transformer-js": "2.14.0",
+    "@parcel/codeframe": "2.14.0",
```

Comment:
?

Reply:
we wind up with duplicates for some reason, it's the weirdest thing, even though 2.14 fulfills ^2.13.1... yarn decides it should be 2.15.1. So I locked down the ones I found to be getting strange duplicates
```
├─ @parcel/config-default@npm:2.14.0 [8657b]
│  └─ @parcel/transformer-js@npm:2.14.0 [ef688] (via npm:2.14.0 [ef688])
│
├─ @parcel/config-storybook@npm:0.0.2
│  └─ @parcel/transformer-js@npm:2.15.1 [144cf] (via npm:^2.13.1 [144cf])
```